src/VisualStudio/Core/Next/Remote/JsonRpcSession.cs

```diff
@@ -16,87 +16,35 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal class JsonRpcSession : RemoteHostClient.Session
```

Comment:
I can rename it as ServiceHub session. this abstraction exist since framework is in Workspace, and service hub concept only exist in VS layer.